packages/apktool.vm/apktool.vm.nuspec

```
@@ -6,8 +6,8 @@
     <authors>Connor Tumbleson, Ryszard Wisniewski</authors>
     <description>A tool for reverse engineering 3rd party, closed, binary Android apps.</description>
     <dependencies>
-      <dependency id="common.vm" />
-      <dependency id="apktool" version="[2.9.1]" />
+      <dependency id="common.vm" version="0.0.0.20240104" />
```

This version doesn't exist:
```suggestion
      <dependency id="common.vm" version="0.0.0.20231221" />
```